code/obj/item/flamethrower.dm

```diff
@@ -2,10 +2,20 @@
 CONTAINS:
 GETLINEEEEEEEEEEEEEEEEEEEEE
 (well not really but it should)
+A Flamethrower
+A Flamethrower that uses a backpack fuel/gas tank
+A Flamethrower backpack fuel/gas tank
+A Flamethrower in various states of assembly
 
 */
-
-/obj/item/flamethrower/
+#define FLAMER_DEFAULT_TEMP 700
+#define FLAMER_MIN_TEMP T0C
+#define FLAMER_MAX_TEMP 1000
```

Comment:
CELCIUS? KELVIN?

Reply:
kelvius, of course.

changed it to 1000C